<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidgetClient">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>771</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_Carte">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Carte fidélité</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Client">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>260</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="styleSheet">
      <string notr="true">#page_4 {
background-color: transparent;
border-image: url(:background.png);
background: none;
border: none;
background-repeat: none;
}</string>
     </property>
     <widget class="QPushButton" name="consulter_CF">
      <property name="geometry">
       <rect>
        <x>142</x>
        <y>247</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>consulter les cartes fidelitée</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gerer_CF">
      <property name="geometry">
       <rect>
        <x>372</x>
        <y>247</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>gerer les cartes fidelitée</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <property name="styleSheet">
      <string notr="true">#page_5 {

border-image:url(:/background.png)
}
</string>
     </property>
     <widget class="QPushButton" name="ajouter_CF">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter carte fidelitée</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_CF">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>260</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier carte fidelitée</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer_CF">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>260</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer carte fidelitée</string>
      </property>
     </widget>
     <widget class="QPushButton" name="re_2_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <property name="styleSheet">
      <string notr="true">#page_6 {

border-image:url(:/background.png)
}
</string>
     </property>
     <widget class="QPushButton" name="consulter_client">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>270</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>consulter les clients</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gerer_client">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>270</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>gerer les clients</string>
      </property>
     </widget>
     <widget class="QPushButton" name="re_3_0">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>132</x>
        <y>260</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter un client </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier un client </string>
      </property>
     </widget>
     <widget class="QPushButton" name="supp_client">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>suprrimer un client </string>
      </property>
     </widget>
     <widget class="QPushButton" name="re_4_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QLineEdit" name="id_client">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="age_client">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_client">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="adresse">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom_client">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>320</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>adresse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter_client">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>470</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="annuler_client">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>470</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>annuler</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_9">
     <widget class="QLineEdit" name="ident_clie_supp">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>donner l'identifiant a supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer_client">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>210</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>poubelle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="re_6_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_10">
     <widget class="QTableView" name="affichage_client">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>161</y>
        <width>701</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="re_7_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableView" name="tableView_client">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>131</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supp">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>140</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Ajouter">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
